server/model.js

```diff
@@ -0,0 +1,104 @@
+require('dotenv').config();
+const { Client } = require('pg');
+
+const client = new Client({
+  user: process.env.DB_USER,
+  host: process.env.DB_HOST,
+  database: process.env.DB,
+  password: process.env.DB_PW,
+  port: process.env.DB_PORT,
+});
+client.connect();
+
+const funcs = {
+  q: {
+    getAll: (p_id, count, offset) => {
+      const sql = `SELECT question_id,question_body,question_date,asker_name,reported,question_helpfulness
+                   FROM questions
+                   WHERE product_id = $1
+                   AND reported = ${false}
+                   ORDER BY question_helpfulness DESC, question_id ASC
+                   LIMIT $2 OFFSET $3`;
+      return client.query(sql, [p_id, count, offset]);
+    },
+    insert: ({
+      product_id, name, email, body,
+    }) => client.query('SELECT MAX(question_id) from "questions"')
```

Comment:
Couldn't you omit the question ID and have the SERIAL ID advance the ID number for you?